types/change-case-object/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for change-case-object 2.0
+// Project: https://github.com/BinaryThumb/change-case-object#readme
+// Definitions by: James Chao <https://github.com/jameschao>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function camelCase(s: string): string;
+export function camelCase<T>(arr: ReadonlyArray<T>): ReadonlyArray<T>;
```

Comment:
If you pass an array of objects, the result will not be equivalent. Example ([RunKit](https://runkit.com/demensky/5de9bc1d3d6662001a19a5c8)):

```js
const changeCaseObject = require("change-case-object");

changeCaseObject.camelCase([{a_b_c: 'e_f_g'}]); // [{aBC: 'e_f_g'}]
```

I suggest this option:

```suggestion
export function camelCase(arr: ReadonlyArray<string>): ReadonlyArray<string>;
export function camelCase(arr: ReadonlyArray<object>): ReadonlyArray<object>;
```

Reply:
👍